### ISSUE TITLE
`aws_route_not_specified_target`: missing `core_network_arn` as target

### ISSUE THREAD

**Issue by ttretau** [terraform-linters/tflint-ruleset-aws#483]
Seems that `aws_route_not_specified_target` check is missing `core_network_arn` as valid target.

```hcl
resource "aws_route" "foo" {
	route_table_id = "rtb-1234abcd"
	cidr_block = "10.42.0.0/16"
	core_network_arn = "arn:aws:networkmanager::230703758040:core-network/core-network-0ce39423cf52b4c76"
}
```
fails with 

```
Error: The routing target is not specified, each aws_route must contain either egress_only_gateway_id, gateway_id, instance_id, nat_gateway_id, network_interface_id, transit_gateway_id, vpc_peering_connection_id or vpc_endpoint_id. (aws_route_not_specified_target)
```

See terraform provider [Argument reference](https://registry.terraform.io/providers/hashicorp/aws/4.65.0/docs/resources/route#argument-reference).
